### PR TITLE
Demo of the Drawer

### PR DESCRIPTION
**Pull request by keemor**
Demo of the Drawer

### CODE REVIEW

File: docs/src/routes/drawer/index.js [hunk 1]
```diff
@@ -83,7 +85,32 @@ export default class DrawerPage extends Component {
         </div>
 
         <div className="mdc-typography--display1">Demo </div>
-        <div className="mdc-typography--title">not available </div>
+        <Button
```
Comment (prateekbh) [inline docs/src/routes/drawer/index.js]: can you try to contain the drawer into a smaller container div?

Comment (keemor) [reply docs/src/routes/drawer/index.js]: What should be the effect of this change?
Now the result is like on this example: http://www.material-ui.com/#/components/drawer
and I followed this example https://material.preactjs.com/component/dialog

Comment (prateekbh) [reply docs/src/routes/drawer/index.js]: i am not able to find a live example, but consider something like a drawer opening/closing inside a div viewport

Comment (keemor) [reply docs/src/routes/drawer/index.js]: I added drawer toggling inside div

**Pull request merged**